examples/grafana.build.sh

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env bash
+
+#
+# A rkt container pod for running a Statsd/Graphite/Grafana metrics stack.
+#
+# Start the containers with new data storage directory:
+#
+#   mkdir data
+#   sudo rkt run \
+#     --port=carbon:2003 \
+#     --port=statsd:8125 \
+#     --port=graphite-api:8888 \
+#     --port=grafana-www:3000 \
+#     --volume data,kind=host,source=$(pwd)/data \
+#     woofwoofinc.dog/grafana-carbon --mount volume=data,target=/opt/graphite/storage/whisper \
+#     woofwoofinc.dog/grafana-statsd \
+#     woofwoofinc.dog/grafana-graphite-api --mount volume=data,target=/srv/graphite/whisper \
+#     woofwoofinc.dog/grafana-www --mount volume=data,target=/usr/share/grafana/data
+#
+# Create some test metric data from the macOS host:
+#
+#    brew install watch
+#    watch -n5 'echo "1d6:$(jot -r 1 1 6)|g" | nc -u -w0 rktmachine.local 8125'
+#
+# This metric can be inspected in a browser on the macOS host:
+#
+#    open 'http://rktmachine.local:8888/render?target=stats.gauges.1d6&from=-5min'
```

Comment:
This just saved me from googling stupid things in order to find the right URLs to poke my `graphite-api` service with. Thanks!